pages/signinLinkSent.js

```diff
@@ -48,6 +49,9 @@ class SignInLinkSent extends Component {
           <P color="black.700" mt={3}>
             You&apos;ll be automatically redirected to the page before signing in. You can close this tab.
           </P>
+          <P mt={3}>
+            <OpenEmailProviderButton email={email} />
+          </P>
```

Comment:
This will add a margin even when there's no button. Let's have the `mt` (marginTop`) or `my` (margin Y) on the button directly to avoid that.